### PR TITLE
Global styles adjustments

### PR DESCRIPTION
**Pull request by aju**
- update font from extra bold to bold
- change primary gray color
- update font size small size

screen shot 2016-01-11 at 11 27 21
screen shot 2016-01-11 at 11 27 33
screen shot 2016-01-11 at 11 27 49
screen shot 2016-01-11 at 11 27 52

### CODE REVIEW

File: src/sass/_fonts.scss [hunk 1]
```diff
@@ -11,7 +11,7 @@ $includeHtml: false !default;
 
   @font-face {
     font-family: $fontFamilyProximaNova;
-    src: url($mintFontsPath + 'ProximaNova-Extrabold.woff') format('woff');
+    src: url($mintFontsPath + 'ProximaNova-Bold.woff') format('woff');
```
Comment (vergilius) [inline src/sass/_fonts.scss]: we have this font in our set?

**Comment by vergilius** [comment]
:ok:

**Comment by matzimowski** [comment]
extra(bold)!
:ok_woman:

**Comment by kdzwinel** [comment]
Remove gray alt.

**Comment by vergilius** [comment]
:+1:

**Comment by kdzwinel** [comment]
LGTM

**Comment by aju** [comment]
Migration:
replace: `$grayAlt` with ` $grayPrimary`
replace: `mint-icon--gray-alt` with `mint-icon--gray`

**Pull request merged**